PWGCF/DataModel/DptDptFiltered.h

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+#ifndef O2_ANALYSIS_DPTDPTFILTERED_H
+#define O2_ANALYSIS_DPTDPTFILTERED_H
+
+#include "Framework/ASoA.h"
+#include "Framework/AnalysisDataModel.h"
+
+namespace o2
+{
+namespace aod
+{
+/* we have to change from int to bool when bool columns work properly */
+namespace dptdptfilter
+{
+DECLARE_SOA_COLUMN(EventAccepted, eventaccepted, uint8_t); //! If the collision/event has been accepted or not
```

Comment:
To my understanding `bool` is OK now (see TrackSelection.h`)

Reply:
Yeah, this has been in this shape from the beginning 
If now it is supported I will modify it then

Reply:
`TrackSelection` table is still with `uint8_t`
Anyway, I will move it to `bool` and check if it works

Reply:
Yes, you are right. Anyhow it should work now. We should also then change the TrackSelection ones. Let me know if it works for you.

Reply:
It worked nice!